### ISSUE TITLE
The skip flag is broken for commands

### ISSUE THREAD

**Issue by vkrol** [Arkweid/lefthook#93]
Minimal reproducible example: https://github.com/vkrol/lefthook-broken-skip-for-commands
1. Run `lefthook install`
2. Run `lefthook run pre-commit`
3. `123` was printed, but it wasn't supposed to:
```
RUNNING HOOKS GROUP: pre-commit
EXECUTE > foo
(SKIP BY SETTINGS)
SUMMARY: (SKIP EMPTY)
123       
```

I also noticed that the check of the flag (https://github.com/Arkweid/lefthook/blob/master/cmd/run.go#L228) occurs after the command is called (https://github.com/Arkweid/lefthook/blob/master/cmd/run.go#L216). Is this correct?

**Comment by MhMadHamster** [comment]
I think the same problem persists for scripts as well. You can easily check it by adding script that will echo smth in a long loop and add `skip: true` for the script in question. It seems like both scripts and commands executes without considering skip flag at all, and the flag only determines whether stdout will be forwarded to the client or not.

**Comment by Arkweid** [comment]
@vkrol yea, you right. The process will be closed immediatyly(because of defer after return), but really, there's no point starting a process at all. Just need to move it above. This is a bug. I will fix it :+1:

**Comment by Arkweid** [comment]
Fixed in 0.6.7
